installer/scripts/omsadmin.sh

```diff
@@ -209,12 +198,6 @@ parse_args()
             ONBOARDING=1
             URL_TLD=$OPTARG
             ;;
-        b)
```

Comment:
please also remove the 'b' and 'r' option in the parameter of getopts. https://github.com/Microsoft/OMS-Agent-for-Linux/pull/296/files#diff-9a868193b8ffdccfa122caa1ff6a6620R184